spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -103,20 +99,7 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
         var link = linksToTest[i]
         GOVUK.triggerEvent(link, 'click')
 
-        expected.event_data.url = link.getAttribute('href')
-        expected.event_data.text = link.innerText.trim()
-        expect(window.dataLayer[0]).toEqual(expected)
-      }
-    })
-
-    it('detects external click events on missing http/https external links', function () {
```

Comment:
Just wanted to check that this bit was deleted on purpose - it's not immediately clear to me what's changed here.

Reply:
@andysellick I deleted it because they were testing broken links - if the link doesn't have `http://` or `https://` or `//` at the start it's treated as an internal link. For example `href="google.com` on our homepage, JS resolves it to be  `https://www.gov.uk/google.com` - so this test was wrong as links looking like that wouldn't exist

Reply:
Ah okay, thanks.